.pre-commit-config.yaml

```diff
@@ -3,22 +3,49 @@ repos:
     hooks:
       - id: web-js-lint-check
         name: Web JS Lint Check
-        entry: bash -c 'make -j$(nproc) -C webui/react check-eslint check-prettier-js'
+        entry: >
+          bash -c
+          '
+          cd webui/react &&
+          FILES=$(
+            git diff --name-only --staged --relative src/**/*.js src/**/*.jsx src/**/*.ts src/**/*.tsx |
```

Comment:
Wanted to combine all file extensions like `src/**/*.{ts.tsx,js,jsx}` but got git error

Reply:
Did you double-check the outputs of this?  I don't think bash `**/*.js` works like zsh's `**/*.js` works.

If I run `bash -c 'echo **/*.js'` from the root of the project I get no files at all.

Reply:
Yse i double-checked. It works fine with the command on Mac.

Reply:
wait... if you do `cd /path/to/determined && bash -c 'echo **/*.js'`, you mean you _don't_ get an empty response on mac?